### DISCUSSION TITLE
Using Tags

### DISCUSSION THREAD

**Discussion by Yahya0007** [shinyorg/shiny#1386] [Q&A]
Hi

I am trying to use Tags as par docs.

          var push = Host.Current.Services.GetService<Shiny.Push.IPushManager>();
          var supported = push.IsTagsSupport();
          if (supported)
          {
              await push.TryRequestAccessWithTags("all");
          }

For both IsTagsSupport and TryRequestAccessWithTags its say 'IPushManager does not contain definition of ..'

What am I doing wrong please?

Thanks

Regards

**Comment by aritchie** [comment]
Ok listen, please spend some time looking at the source as well before just dropping in a support request.  I have updated the docs due to this and I've added back the extensions.  Please realize this is not a product.  Do your own research.  The code is free!
